rocAL/rocAL/include/image_loader.h

```diff
@@ -71,10 +76,10 @@ class ImageLoader : public LoaderModule {
     bool _is_initialized;
     bool _stopped = false;
     bool _loop;//<! If true the reader will wrap around at the end of the media (files/images/...) and wouldn't stop
-    const static size_t CIRC_BUFFER_DEPTH = 3; // Used for circular buffer's internal buffer
+    size_t CIRC_BUFFER_DEPTH; // Used for circular buffer's internal buffer
```

Comment:
if this is not constant, then it is better to rename it. (_prefetch_queue_depth).